core/templates/domain/collection/CollectionRightsObjectFactory.ts

```diff
@@ -109,12 +108,15 @@ export class CollectionRights {
   // internal, bindable objects are changed within this collection rights.
   // Note that the collection nodes within this collection will be completely
   // redefined as copies from the specified collection rights.
-  copyFromCollectionRights(otherCollectionRights: CollectionRights): void {
-    this._collectionId = otherCollectionRights.getCollectionId();
-    this._canEdit = otherCollectionRights.canEdit();
-    this._isPrivate = otherCollectionRights.isPrivate();
-    this._canUnpublish = otherCollectionRights.canUnpublish();
-    this._ownerNames = otherCollectionRights.getOwnerNames();
+  static copyFromCollectionRights(
```

Comment:
Rewrite/remove the comment above.

Reply:
this comment need no rewrite I think, do let me know if I'm wrong.

Reply:
I would rewrite it to smth like 
```
  // Create copy of the collection rights. This is performed as a deep copy.
```

Reply:
done! @vojtechjelinek, is this fine now?

Reply:
Sorry for getting to you a bit late. I have added my suggestion above.